### DISCUSSION TITLE
OpenShift/OKD manifests

### DISCUSSION THREAD

**Discussion by se35710** [ScoopInstaller/Main#3055] [Show and tell]
I setup a while back a [separate repo for OpenShift/OKD manifests](https://github.com/se35710/scoop-redhat), but now when Main/Extras/Versions are in a better state, maybe those manifests from my repo should be moved there?

**Comment by rashil2000** [comment]
I think many of them already exist in Main? I remember reviewing them.

**Reply by se35710** [reply]
I can only find two ([openshift-okd-client.json](https://github.com/ScoopInstaller/Main/blob/master/bucket/openshift-okd-client.json) and [openshift-origin-client.json](https://github.com/ScoopInstaller/Main/blob/master/bucket/openshift-origin-client.json) which are the same thing; OKD === origin), but no "normal" OpenShift CLI, nor the related tools, for example Podman.

**Reply by rashil2000** [reply]
Okay, please make PRs for them
